src/charm.py

```diff
@@ -326,7 +322,7 @@ def _loki_command(self):
         return f"/usr/bin/loki -config.file={LOKI_CONFIG}"
 
     @property
-    def _build_pebble_layer(self) -> Layer:
+    def _loki_pebble_layer(self) -> Layer:
```

Comment:
I changed this because it is more clear than `build` since that makes it sound like a method not a property. Is there any other location that I should refactor this besides within the `src/charm.py` file?

Reply:
other than test code I wouldn't expect this to be accessed from anywhere else. If the tests pass you're good.